src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/msg/ProcessedEnvelope.java

```diff
@@ -12,14 +12,14 @@
 
     public final UUID envelopeId;
 
-    public final Long ccdId;
+    public final String ccdId;
 
     public final String envelopeCcdAction;
 
     public ProcessedEnvelope(
         @JsonProperty("id") UUID id,
         @JsonProperty("envelope_id") UUID envelopeId,
-        @JsonProperty("ccd_id") Long ccdId,
+        @JsonProperty("xx_ccd_id") String ccdId,
```

Comment:
is that prefix intentional?

Reply:
yes. as orchestrator sending this field it may cause conversion problem

Reply:
unknown field are ignored